### ISSUE TITLE
Multisite: Global settings

### ISSUE THREAD

**Issue by flowdee** [WPManageNinja/fluent-smtp#37]
Would be great if there's a way to enable the plugin network-wide inside a multisite and when doing so, defining SMTP credentials which will be taken for all sites in the network by default.

Right now, I have to configure the plugin inside every single multisite blog :-(

**Comment by JDPerf** [comment]
Agreed, this is still a major incompatibility with the current version since if it's network activated then the primary site settings should _always_ apply to all of the sites of the multisite network. If not network activated then individual site configurations are fine.

**Comment by buegelfrei** [comment]
As of Feb 26, 2025, I was able to configure one set of credentials for an entire multisite network by activating the plugin on the main domain site only. To the extent I have tested, the subdomain sites are sending emails using the main site's settings.
